app/src/main/java/chat/rocket/android/dagger/module/AppModule.kt

```diff
@@ -331,4 +366,33 @@ class AppModule {
     ): DatabaseManager {
         return factory.create(currentServer)
     }
+
+    @Provides
+    @Singleton
+    fun provideAnswersAnalytics(): AnswersAnalytics {
```

Comment:
You can use `@IntoSet` and later on `provideAnalyticsManager` use a `Set<Analytics>` to receive all implementations

Reply:
Thanks for letting me know about this another possibility. 👐